README.md

```diff
@@ -83,28 +85,54 @@ development IDE - see below) rather than using brew.
       See [Elastic search website](https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html)
       Just pull the image to install. See later for how to run.
 
+
 - Kibana (for monitoring Elasticsearch)
     - Install Docker image.
       See [Elastic search website](https://www.elastic.co/guide/en/kibana/current/docker.html)
       Just pull the image to install. See later for how to run.
 
+
 - Git - [see Git website](https://git-scm.com/downloads)
-- PostgreSQL - [Postgres website](https://www.postgresql.org/download/)
+
+
+- PostgreSQL 
```

Comment:
FWIW, I ended up using Docker for this (since I already had it for ES/Kibana):

- `docker pull postgres:14`
- `docker run -d --name postgres -e POSTGRES_PASSWORD=postgres -p 5432:5432 library/postgres:14`

I'd imagine that'd get around the M1 problems.

Reply:
Neat idea! However, in development we find it useful to be able the play around with the dev database using IntelliJ's built in DB tools. Do you know if Intellij can access a data source inside a Docker image?

Reply:
> Do you know if Intellij can access a data source inside a Docker image?

Absolutely! I do it myself. Just connect to localhost port 5432 as if you were running it locally.